src/pages/docs/[source]/[tag]/search.vue

```diff
@@ -2,6 +2,53 @@
 	<div class="mx-auto py-16 px-4 sm:px-8 lg:py-8 w-full">
 		<div class="prose prose-discord dark:prose-light mx-auto max-w-4xl lg:max-w-full">
 			<h1>Search Results</h1>
+			<div class="prose dark:prose-light flex flex-col md:flex-row">
```

Comment:
Are the `prose dark:prose-light` here necessary? Considering it cascades from the `div` block above

Reply:
For some reason it didn't cascade when I was testing this, but I just tried it and now it does 😓